packages/bff/src/model/mappers.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  agreementApi,
+  catalogApi,
+  tenantApi,
+} from "pagopa-interop-api-clients";
+import {
+  agreementApiState,
+  catalogApiDescriptorState,
+} from "./api/apiTypes.js";
+
+/* 
+  This file contains commons utility functions 
+  used to pick or transform data from model to another.
+*/
```

Comment:
Proposal: let's rename the file into `modelUtils.ts` or `modelMappingUtils.ts` or even better `apiConverterUtils.ts` - in this way this comment becomes unnecessary, the name makes it clear

Reply:
make sense